pkg/resources/schema_acceptance_test.go

```diff
@@ -31,36 +97,283 @@ func TestAcc_Schema(t *testing.T) {
 		CheckDestroy: acc.CheckDestroy(t, resources.Schema),
 		Steps: []resource.TestStep{
 			{
-				ConfigDirectory: config.TestNameDirectory(),
-				ConfigVariables: map[string]config.Variable{
-					"name":     config.StringVariable(name),
-					"database": config.StringVariable(acc.TestDatabaseName),
-					"comment":  config.StringVariable(comment),
+				PreConfig: func() {
+					params := acc.TestClient().Parameter.ShowAccountParameters(t)
+					*accountDataRetentionTimeInDays = helpers.FindParameter(t, params, sdk.AccountParameterDataRetentionTimeInDays).Value
```

Comment:
maybe try using the new object parameters assertions here? 🙏

Reply:
If we have time before release, then sure.